lib/assert.js

```diff
@@ -89,7 +56,8 @@ function fail(actual, expected, message, operator, stackStartFunction) {
     message = actual;
   if (arguments.length === 2)
     operator = '!=';
-  throw new AssertionError({
+  const errors = lazyErrors();
+  throw new errors.AssertionError({
```

Comment:
If I remember, there was a problem caused by cyclic dependency. Something wasn't initialized causing few test cases to fail. Let me check it again!